test/test_helper.rb

```diff
@@ -57,6 +57,16 @@ def new_ghostferry(filename, config: {})
     g
   end
 
+  def new_ghostferry_with_termination_after_row_copy_completed(filename, config: {})
```

Comment:
Minor nitpick: if there's no other tests that uses this method, might as well inline it so people who read it don't have to chase this function to a separate file?

Reply:
Addressed in 766ef53: I'll clean up the history before I merge.